database/migrations/wallet-migration.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+      await queryInterface.createTable('wallets', 
+      {
+        id: {
+          type: Sequelize.UUID,
+          primaryKey: true,
+        },
+        user_id: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+        status: {
+            type: Sequelize.ENUM('Active', 'Inactive'),
```

Comment:
import enum variables from the constants.js file and use them here instead of using a string literal. Helps to maintain uniformity. you can find the file in src/utils/constants.js